src/API.js

```diff
@@ -0,0 +1,33 @@
+// https://codesandbox.io/s/network-request-practice-forked-with-changes-cfs77
+
+
+export function getTraveler(id) {
```

Comment:
This looks great. You could create a reusable function her that dynamically adds the specific endpoint, but don't worry about this now.